### PR TITLE
s3deploy 1.0.8 (new formula)

### PR DESCRIPTION
**Pull request by bep**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/s3deploy.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  url "https://github.com/bep/s3deploy/releases/download/v1.0.8/s3deploy_1.0.8_macOS-64bit.tar.gz"
+  version "1.0.8"
+  sha256 "c6d47f0635bbd821553e72414c49dd767df980622112e23e81a9edd54c3b7499"
+
+  def install
+    bin.install "s3deploy"
+  end
+
+  test do
+    system "#{bin}/goreleaser", "-h"
```
Comment (tchajed) [inline Formula/s3deploy.rb]: Did you mean to test `s3deploy` here?

**Review by tchajed — CHANGES_REQUESTED** [review]
Thanks for your contribution to Homebrew!

It looks like the test for this formula is failing. Once you've fixed it, you can test locally with `brew test s3deploy`. (The contributing guidelines don't mention this for new formulae but it is a requirement for the continuous integration - sorry!)

File: Formula/s3deploy.rb (hunk 1 above)
Comment (bep) [inline Formula/s3deploy.rb]: Oh, sorry ... Copy and paste error ... I will fix.

File: Formula/s3deploy.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+class S3deploy < Formula
+  desc "Deploy static websites to Amazon S3 with Gzip and headers support."
+  homepage "https://github.com/bep/s3deploy"
+  url "https://github.com/bep/s3deploy/releases/download/v1.0.8/s3deploy_1.0.8_macOS-64bit.tar.gz"
```
Comment (ilovezfs) [inline Formula/s3deploy.rb]: This needs to be built from source.

**Review by ilovezfs — CHANGES_REQUESTED** [review]

**Comment by ilovezfs** [comment]
Feel free to open a new PR once this is actually building from source.

**Pull request closed without merging**